index.html

```diff
@@ -239,6 +239,25 @@ <h3 class="knowledge-title poppins-font-extrabold">
                 </article>
             </div>
         </section>
+        <section id="form-m" class="form-mobile full-width">
+            <div class="header-image full-width"></div>
+            <div class="form-header">
+                <h1>Get Started</h1>
+                <p class="poppins-font-regular">If you are interested in hiring my services, fill out the following form and I will contact you as soon as possible.</p>
+            </div>
+            <form class="contact-form" action="https://formspree.io/f/mwkzgbvy" method="post">
+                <fieldset class="input-fieldset">
+                    <label for="full-name">Your name</label>
+                    <input type="text" name="full-name" id="full-name" placeholder="Full name" required maxlength="30" class="full-width poppins-font-regular">
+                </fieldset>
+                <fieldset class="input-fieldset">
+                    <label for="email">Your email</label>
+                    <input type="email" name="email" id="email" placeholder="example@example.com" required class="full-width poppins-font-regular">
+                </fieldset>
+                <textarea class="textarea full-width" required type="textarea" id="message" name="message" placeholder="Enter your message here..." maxlength="500"></textarea>
+                <button type="submit" class="main-btn full-width">Start Collaboration</button>
+            </form>
+        </section>
```

Comment:
- [ ] Great work implementing 95% of the features. Your collaboration speaks for itself. 👏 I noticed the `background-color` for the footer is `#F9F9FA` but yours is `white`. Kindly make this change so that your design is identical to that of the Figma template as much as possible. 💻 
- [ ] The second change to make has to do with the social-media icons you omitted in the design. In the template, the social media icons, the `Get My Resume` and the icon is missing from your design. Kindly include them in your design. 👍  

### Your Design

![image](https://user-images.githubusercontent.com/77106962/206043513-1f97267f-1a0c-4780-9896-c506ddfa22e6.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/206043612-cbc6fbf2-0e48-4c09-9ce2-f0b890393faa.png)